custom/eqa/expressions.py

```diff
@@ -16,7 +16,13 @@
 
 
 def get_val(form, path, default=0):
-    return int(form.get_data(path)) if form else default
+    if not form:
+        return default
+    question_value = form.get_data(path)
+    try:
+        return int(question_value)
+    except ValueError:
```

Comment:
note this can also raise TypeError, for example, if question_value is None. not sure if that applies here or not (if question_value could potentially be None or some object type)